pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadata/ownership/requirement.go

```diff
@@ -52,14 +52,22 @@ func generateViolationJSON(pkiInfo *certgraphapi.PKIRegistryInfo) *certgraphapi.
 
 	for i := range pkiInfo.CertKeyPairs {
 		curr := pkiInfo.CertKeyPairs[i]
-		owner := curr.CertKeyInfo.OwningJiraComponent
+		certKeyInfo, err := tlsmetadatainterfaces.CertInfoForCertKeyPair(curr)
+		if err != nil {
+			continue
+		}
+		owner := certKeyInfo.OwningJiraComponent
 		if len(owner) == 0 || owner == tlsmetadatainterfaces.UnknownOwner {
 			ret.CertKeyPairs = append(ret.CertKeyPairs, curr)
 		}
 	}
 	for i := range pkiInfo.CertificateAuthorityBundles {
 		curr := pkiInfo.CertificateAuthorityBundles[i]
-		owner := curr.CABundleInfo.OwningJiraComponent
+		caBundleInfo, err := tlsmetadatainterfaces.CertificateAuthorityInfoForCABundle(curr)
```

Comment:
don't like this block with an error.  We know the metadata at this point.  I resolved with https://github.com/openshift/origin/pull/28539/files#diff-b2792dc75c2beb82a9699af3c941303e12704132ebf6a6f102ccc7069a408a7dR18